.gitignore

```diff
@@ -4,12 +4,16 @@
 
 # Generated by gradle
 *.iml
+*.gitignore
 
 # User-specific stuff:
 .idea/workspace.xml
 .idea/tasks.xml
 .idea/dictionaries
 .idea/sonarIssues.xml
+.idea/misc.xml
+.idea/modules.xml
+.idea/vsc.xml
```

Comment:
These files should not be ignored as they contain project-specific configuration that should be shared across all developers

Reply:
Probably my ide changed it and I did not notice